packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -1261,7 +1224,7 @@ export class LitNodeClientNodeJs extends LitCore {
         return;
       }
 
-      const signature: any = combineEcdsaShares(sigShares);
+      const signature: any = combineEcdsaShares(shares);
```

Comment:
```suggestion
      const signature: CombinedECDSASignature = combineEcdsaShares(shares);
```

Reply:
Or just remove the `any` and let TS infer it

Reply:
`CombinedECDSASignature` is not a complete for this object. We should update it but unsure if it will break other implementations. for now going to let the type be inferred.